### PR TITLE
Misc doc improvements for default BUILD file for nixpkgs_package

### PR DESCRIPTION
**Pull request by googleson78**

### CODE REVIEW

File: core/nixpkgs.bzl [hunk 1]
```diff
@@ -336,9 +336,9 @@ def nixpkgs_package(
           <dt><code>:bin</code></dt>
           <dd>Everything in the <code>bin/</code> directory.</dd>
           <dt><code>:lib</code></dt>
-          <dd>All <code>.so</code> and <code>.a</code> files that can be found in subdirectories of <code>lib/</code>.</dd>
+          <dd>All <code>.so</code>, <code>.dylib</code> and <code>.a</code> files that can be found in subdirectories of <code>lib/</code>.</dd>
           <dt><code>:include</code></dt>
-          <dd>All <code>.h</code> files that can be found in subdirectories of <code>bin/</code>.</dd>
+          <dd>All <code>.h</code>, <code>.hh</code>, <code>.hh</code>, <code>.hpp</code> and <code>.hxx</code> files that can be found in subdirectories of <code>include/</code>.</dd>
```
Comment (aherrmann) [inline core/nixpkgs.bzl:341]: `.hh` is duplicated.

**Review by aherrmann — COMMENTED** [review]
CI is read, perhaps `core/README.md` needs to be regenerated, too?

File: core/nixpkgs.bzl (hunk 1 above)
Comment (googleson78) [reply core/nixpkgs.bzl]: Removed it, thanks.

**Review by aherrmann — APPROVED** [review]
Thanks!

**Pull request merged**